extensions/dicom-seg/src/components/SegmentationPanel.js

```diff
@@ -0,0 +1,552 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
This is very clean code, my only concern is that it may be a bit long, perhaps some of the component should be split up into seperate files?